src/main/java/gregtech/api/recipes/crafttweaker/MetaItemBracketHandler.java

```diff
@@ -6,34 +6,58 @@
 import crafttweaker.api.item.IItemStack;
 import crafttweaker.mc1120.item.MCItemStack;
 import crafttweaker.zenscript.IBracketHandler;
+import gregtech.api.items.materialitem.MaterialMetaItem;
 import gregtech.api.items.metaitem.MetaItem;
 import gregtech.api.items.metaitem.MetaItem.MetaValueItem;
+import gregtech.api.unification.OreDictUnifier;
+import gregtech.common.items.MetaItems;
```

Comment:
This coupling is wrong. We should not access GTCE common from API. Please look for other way how to do it.